journal.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Digital Dr3am T3am - Journal Entry Page</title>
+</head>
+
+<body>
+    <!-- Page Navigation -->
+    <head>
+        <!-- Date -->
+        <section id="date">
+            <button type="button"><</button>
+            <h2>May 15, 2024</h2>
+            <button type="button">></button>
+        </section> 
+
+        <!-- Return to Home -->
+        <section id="home">
+            <button type="button">Home</button>
+        </section> 
+    </head>
+    
+    <main>
+        <!-- Entry Text Editor -->
+        <section id="entry-text-editor">
+            <h2>Journal Entry Title</h2>
+            <textarea id="entry-text" rows="30" cols="50"></textarea>
+        </section>
+
+        <!-- Tasks & Sentiment Rating -->
+        <section id="tasks-sentiment-rating">
+            <!-- Tasks -->
+            <h2>Tasks</h2>
+            <section id="task-list">
+                <button class="new-task" type="button">+ New Task</button>
+            </section>
+            
+            <!-- Sentiment Rating -->
+            <section id="meter-section">
+                <h2>How are you feeling today?</h2>
+                <form class="meter" action="#">
```

Comment:
Looks like this form tag wasn't closed, but I don't think we'll need it anyway so you can remove it along with the submit button. We'll aim to use JS to just have a one-click sentiment rating (so no need to click submit to save)